### ISSUE TITLE
Resonance table from Mythic Charms not working?

### ISSUE THREAD

**Issue by tehPete** [LunaPixelStudios/Better-MC#1478]
### Describe the Bug

The table is constructed correctly, but does not do anything. Items thrown on top swirl around like they're being detected, but right-clicking on the block does nothing and throwing an xp potion on top does not complete the recipe.

### Reproduction Steps

Install update, try using mod.

### Operating System

Win 10

### Minecraft Version

1.20

### Modloader

Fabric

### Modpack Version

BMC v20

**Comment by SHXRKIIE** [comment]
It works, please press shift and read the the ring and table for what you need and after placing the items throw a bottle of enchanting

**Comment by tehPete** [comment]
I just did some more testing to try and figure this out, as I was following the instructions from the mod page to the letter and it was not working. 

I went into my existing Curseforge loader BMC v20 profile (which was updated from v19 using the loader), tried again and it still would not work - right-clicking on the table with the recipie item did nothing, as did Q throwing everything on the table and then right-click throwing a bottle of enchanting on the table. I tried the ring in both applicable trinket slots, as well as in the offhand as shown in the images on the mod page - nothing. The slower pickup of thrown items due to having the ring equipped was working, the items were seemingly being detected by the table when thrown on with Q, but it just would not complete.

From there I created a new profile in the loader for BMC v20, by using the Change Version feature and selecting to update to a new profile. I tested exactly the same recipie in this fresh profile and it worked. Of note, right-clicking the table in the new profile now placed the crafting item on the resonance table as expected - this function was not working in the original profile.

I went back to my original profile, re-tested (I had all crafting items already in my loadout from the previous attempt) and found that it was now working there too - both right-clicking on the table, and the crafting itself.

I then loaded up the server I've been running for the last few weeks and updating with the server packs as new versions come out, and found it was also working there too now exactly as expected.

Prior to creating a new profile, the right-clicking function for the resonance table absolutely did not work. Maybe it's an issue with the CF loader not updating a shared file it should have during the original update from v19 to v20? Whatever the bug is, it seems to be client-based.

I figured I'd post all this here in case anyone else comes across a similar bug in future. At least the fix is a simple one for now!

**Comment by ads-webhook-dummy** [comment]
I have owned xbox 360 edition for many years and eventually moved to ps4 meaning i lived on bedrock edition. Only just now getting into modded java has been a great experience but i realize how many mods clash and cause issues. I assumed the same when i saw the table doesn't work. This was the best I found for a fix and I didn't understand a lick of it. Everything i read just then was gibberish. Is there any way to fix the mod not working without having to do some gymnastics with my puter.
